Tutorial/GettingStartedGuide.md

```diff
@@ -0,0 +1,65 @@
+# Getting Started Guide  
+  
+Creating a rest api to expose model's predict functionality  
+  
+  
+## Table of Contents  
+1. [pre-requisites](#pre-requisites)   
+2. [predict](#predict)  
+3. [api](#api)  
+4. [build-container](#build-container)  
+5. [run-container](#run-container)  
+5. [test-container](#test-container)  
+  
+## pre-requisites  
+  
+1. trained model  
+2. [source-to-image (s2i)](https://github.com/openshift/source-to-image)  
+3. [Httpie](https://httpie.org/) (Required to run `make test`)   
+  
+## predict     
+ Create a predict function that takes a message object as function parameter containing data instances to predict against   
+and return list of predictions   
+
+   > refer to function `predict_iris_type` in `iris_classification/predict.py`
+  
+##  api
+  
+Exposing model predict as an api using [s2i](https://github.com/openshift/source-to-image) requires  
+  
+1. creating a `.s2i/environment` config file in the top-level directory with  
+   * `ROUTES=<routeName:predictModuleName:predictFunctionName>`  
+  > e.g. `ROUTES=iris_classification_svc/predict:predict:predict_iris_type`  
+  
+2. creating a `requirements.txt` to install python-dependencies required to run the model  
+  
+  
+## build-container  
```

Comment:
Few more words :-)

Like:
The following command does blah...   and then put commands in ```code```